app/src/test/java/com/stevesoltys/seedvault/transport/backup/BackupCoordinatorTest.kt

```diff
@@ -72,6 +72,22 @@ internal class BackupCoordinatorTest : BackupTest() {
         requiresNetwork = false
     )
 
+    @Test
+    fun `isAppEligibleForBackup() exempts plugin provider and blacklisted apps`() {
+
+        every {
+            packageService.shouldIncludeAppInBackup(packageInfo.packageName)
+        } returns true andThen false andThen true
+
+        every {
+            plugin.providerPackageName
+        } returns packageInfo.packageName andThen "new.package" andThen "new.package"
+
+        assertTrue(backup.isAppEligibleForBackup(packageInfo, true))
```

Comment:
This was assertFalse before. Why has it changed and is this test still in the spirit of its name?

Reply:
Moved and updated this test to an integration test because there are problems unit testing `PackageService`.